docker-compose-debug.yml

```diff
@@ -0,0 +1,91 @@
+version: "3"
```

Comment:
Everything in this file is the same as the original docker-compose except for things I changed which I commented on.